src/main/index.js

```diff
@@ -997,7 +997,21 @@ function runApp() {
           { role: 'zoomout' },
           { role: 'zoomout', accelerator: 'CmdOrCtrl+numsub', visible: false },
           { type: 'separator' },
-          { role: 'togglefullscreen' }
+          { role: 'togglefullscreen' },
+          { type: 'separator' },
+          {
+            label: 'History',
+            accelerator: 'CmdOrCtrl+U',
```

Comment:
It wouldn't be too hard to have a separate shortcut on macOS and have a meaningful one on Windows and Linux
```suggestion
            accelerator: process.platform === 'darwin' ? 'Cmd+Z' : 'Ctrl+H',
```

Reply:
`Cmd+Z`? `Cmd+Y`?

Reply:
What's the standard shortcut for history in apps on macOS?

Reply:
`Cmd+Y` I guess
- [Safari](https://www.wikihow.com/View-Your-Browser-History-on-a-Mac)
- [Chrome](https://support.google.com/chrome/answer/157179?hl=en)

Reply:
Updated but still need to be tested on windows

Reply:
Thank you for implementing shortcuts! Once a menu item, a shortcut can be customised in macOS, so anything is good with me.

Reply:
> Once a menu item, a shortcut can be customised in macOS, so anything is good with me.

I didn't know that~

Reply:
Yep, like this: 
<img width="780" alt="Screenshot 2022-11-21 at 8 00 25" src="https://user-images.githubusercontent.com/2356354/202985619-b9ff1b5a-7944-49d4-a0f8-2dd91f83ba9e.png">